.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-lint-staged --debug
+yarn lint-staged --debug
```

Comment:
I can omit this. But I thought it odd that the repo requires `lint-staged` be installed globally when it's a dependency of the project.